Core/Debugger/Breakpoints.cpp

```diff
@@ -40,14 +40,15 @@ static std::mutex memCheckMutex_;
 std::vector<MemCheck> CBreakPoints::memChecks_;
 std::vector<MemCheck *> CBreakPoints::cleanupMemChecks_;
 
-void MemCheck::Log(u32 addr, bool write, int size, u32 pc) {
+void MemCheck::Log(u32 addr, bool write, int size, u32 pc, const std::string &reason) {
```

Comment:
Since reason is used as .c_str() only in this function, I'd prefer a plain char pointer here, to avoid a memory allocation. (string_view would be ideal for things like this but is a little too new and not smoothly supported by sprintf) 

There are some similar cases in other parts of this PR.

Reply:
The framebuf and texture ones are dynamic strings, and almost all of these are gonna be 20 bytes or less, so they shouldn't be separate allocations anyway.  I didn't want to have to worry about allocation lifetimes for the dynamic ones, and I definitely don't want to require that these all be static.

-[Unknown]

Reply:
Hm, didn't realize how many of these were dynamic. I'm ok with it now, thanks for changing in memchecks at least.